decidim-core/lib/decidim/attributes/localized_date.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Attributes
+    # Custom Virtus value to parse a String representing a Date using
+    # the app localization format.
+    class LocalizedDate < Virtus::Attribute
+      def coerce(value)
+        return value unless value.is_a?(String)
+        Date.strptime(value, I18n.t("date.formats.decidim_short"))
+      rescue ArgumentError
+        nil
```

Comment:
Very clever @deivid-rodriguez !
Creating a custom attribute coercer is much better design than overriding coercible.
Just a note, why swallow the `ArgumentError`? This will make parsing errors much more difficult to find, I know that they should not happen in production but still when testing they happened to me and was tedious to bind the cause to the specific line.

Reply:
The situation is described [here](https://github.com/decidim/decidim/pull/3724#discussion_r207190768). Basically, we don't want end users to be able to crash our server and their will, so we need to rescue and swallow. We could keep the `nil` return but add some logging in between or even append an error to the attribute, but I think the current version is enough for such an edge case...

Reply:
OK, I understand now. I did not get that the user would provoke a crash when reading your comment before. Indeed raising an error is not what we want.